tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -281,40 +326,12 @@ private void InstrumentationGatewayInstrumentationGatewayEvent(object sender, In
             }
         }
 
-        private bool AreArchitectureAndOsSupported()
-        {
-            var frameworkDescription = FrameworkDescription.Instance;
-            var osSupported = false;
-            var supportedOs = new[] { OSPlatform.Linux, OSPlatform.MacOS, OSPlatform.Windows };
-            if (supportedOs.Contains(frameworkDescription.OSPlatform))
-            {
-                osSupported = true;
-            }
-
-            var archSupported = false;
-            var supportedArchs = new[] { ProcessArchitecture.Arm, ProcessArchitecture.X64, ProcessArchitecture.X86 };
-            if (supportedArchs.Contains(frameworkDescription.ProcessArchitecture))
-            {
-                archSupported = true;
-            }
-
-            if (!osSupported || !archSupported)
-            {
-                Log.Error(
-                    "AppSec could not start because the current environment is not supported. No security activities will be collected. Please contact support at https://docs.datadoghq.com/help/ for help. Host information: operating_system: {{ {OSPlatform} }}, arch: {{ {ProcessArchitecture} }}, runtime_infos: {{ {ProductVersion} }}",
-                    frameworkDescription.OSPlatform,
-                    frameworkDescription.ProcessArchitecture,
-                    frameworkDescription.ProductVersion);
-            }
-
-            return osSupported && archSupported;
-        }
-
         private void RunShutdown()
         {
             if (_instrumentationGateway != null)
             {
-                _instrumentationGateway.InstrumentationGatewayEvent -= InstrumentationGatewayInstrumentationGatewayEvent;
+                _instrumentationGateway.RequestStart -= InstrumentationGatewayInstrumentationGatewayEvent;
```

Comment:
Seems this event is not used in the constructor.

Reply:
thanks good catch, fixed in e4b0527